tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/conversion/NativeConversionWithStampTypesTest.java

```diff
@@ -133,4 +133,9 @@ public void testConvertMapJsonWithDecimalToOpenRecord() {
     public void testConvertMapJsonWithDecimalUnionTarget() {
         BRunUtil.invoke(compileResult, "testConvertMapJsonWithDecimalUnionTarget");
     }
+
+    @Test
```

Comment:
Shall we move these tests to org.ballerinalang.langlib.test.LangLibValueTest#testFromJsonWithType?

Reply:
Fixed.